seaborn/distributions.py

```diff
@@ -465,18 +465,18 @@ def kdeplot(data, data2=None, shade=False, vertical=False, kernel="gau",
     clip : pair of scalars, or pair of pair of scalars, optional
         Lower and upper bounds for datapoints used to fit KDE. Can provide
         a pair of (low, high) bounds for bivariate plots.
-    legend : bool, optinal
+    legend : bool, optional
         If True, add a legend or label the axes when possible.
-    cumulative : bool
+    cumulative : bool, optional
         If True, draw the cumulative distribution estimated by the kde.
-    shade_lowest : bool
+    shade_lowest : bool, optional
         If True, shade the lowest contour of a bivariate KDE plot. Not
         relevant when drawing a univariate plot or when ``shade=False``.
         Setting this to ``False`` can be useful when you want multiple
         densities on the same Axes.
     ax : matplotlib axis, optional
         Axis to plot on, otherwise uses current axis.
-    kwargs : key, value pairings
+    kwargs : key, value pairs
```

Comment:
I was trying to decide whether to use pairings, mappings, or pairs as all three where used in the docs previously, but I agree. Changing to pairings.